*In this tutorial, we will use the tool to input a suspicious business owner and use them to discover other suspicious business owners who use similar addresses and behavoir to register businesses*

## Introduction 

We will begin our journey on Oxford Street in central London, a street that until recently was regarded as the cities premier shopping destination, which over the past two years has seen the opening of a suprising number of American sweet shops to the dislike of many citizens and local authorities. The shops have attracted negative attention for a number of reasons including extortionate prices, out of date/counterfeit products and opaque finances that have led Westminster council to claim they are [overdue £7.9 million](https://www.ft.com/candy) in business rates from 30 sweet shops in the area.

*Kingdom of Sweets* is one store in particular that came under scrutiny recently for unpaid taxes. The [mysterious owner](https://find-and-update.company-information.service.gov.uk/officers/Nd2URspq4bvLy-hwzDZ0_p7FGJw/appointments) of the business has a prolific record of directing many companies that fended off compulsary strike-offs and [failed to pay millions in business rates](https://www.ft.com/content/3d338904-6036-4326-a786-971464907f14). This questionable business owner will serve as the input to our hunt to find other individuals.

![title](assets/candyy.jpeg)

We will begin by using the director of *Kingdom of Sweets* as input. To get other directors, we will follow the following pattern: 

officer -> `get_companies`(officer) -> `get_addresses`(companies) -> `get_companies`(addresses) -> `get_officers`(companies) -> officers

Every officer in Companies House has a unique ID which we can find from their URL. For instance we are interested in this URL: "https://find-and-update.company-information.service.gov.uk/officers/Nd2URspq4bvLy-hwzDZ0_p7FGJw/appointments", and the `officer_id` value is contained in the URL and is: "Nd2URspq4bvLy-hwzDZ0_p7FGJw"

In [1]:
officer_id = "Nd2URspq4bvLy-hwzDZ0_p7FGJw"

API accepts authentication through basic auth where the access_token is `username` and `password` is left blank. We also include a `size` parameter set to the maximum of 5000 to ensure we get as many results from the API as possible. 

In [2]:
from sugartrail import *

When we input the `officer_id`, `get_appointments()` will get all the appointments for the officer and also print some metadata. 

In [3]:
companies = get_appointments(officer_id)

30 appointments
28 active appointments


28 director roles is impressive workload! Now lets check out the correspondance address and registered address for these businesses: 

In [4]:
correspondance_addresses = get_locations(companies,"correspondance")

6 unique postcodes
10 unique correspondance addresses
{'8 Segedunum Business Centre, Station Road, Wallsend, United Kingdom, NE28 6HQ': 8, '220, Consultancy House, Ilford, United Kingdom, IG2 7DY': 7, '220, Consultancy House, Ilford, England, IG2 7DY': 5, 'Burnards Accountants, 8 Segedunum Business Centre, Wallsend, England, NE28 6HQ': 3, '3rd Floor, 207, Regent Street, London, England, W1B 3HH': 2, '3rd Floor, 13 Charles Ii Street, London, England, SW1Y 4QU': 1, '13 Charles Ii Street, 3rd Floor, London, England, SW1Y 4QU': 1, '8 Segedunum Business Centre, Station Road, Wallsend, England, NE28 6HQ': 1, 'Burnard Accountants, 8 Bankside Building, Segedunum Business Centre, Wallsend, United Kingdom, NE28 6HQ': 1, '8 Bankside Bldg Segedunum Business Centre, Station Road, Wallsend, England, NE28 6HQ': 1}


In [5]:
registered_addresses = get_locations(companies, "registered")

3 unique postcodes
8 unique registered addresses
{'220 Consultancy House, Wards Road, England, Ilford, IG2 7DY': 10, '220 Consultancy House, Wards Road, United Kingdom, Ilford, IG2 7DY': 9, '220 Consultancy House Wards Road, Ilford, IG2 7DY': 3, '3rd Floor 13 Charles Ii Street, England, London, SW1Y 4QU': 2, 'Consultancy House, 220 Wards Road, United Kingdom, Ilford, IG2 7DY': 2, 'C/O BDO LLP, 5 Temple Square Temple Street, Liverpool, L2 5RH': 2, '13 Charles Ii Street 3rd Floor, Charles Ii Street, England, London, SW1Y 4QU': 1, '220 Consultancy House, Wards Road, Ilford, IG2 7DY': 1}


We can see our director has many businesses registered to the same addresses. Also a number of these addresses are the same address with alternative details, for instance, in total 25 of businesses are registered to 220 Consultancy House in Ilford which is listed under a number of different variations.  As we can see from the postcodes, there are only 6 unique correspondance addresses and 3 unique registered addresses for the 30 businesses under the director's name. 

![title](assets/consultancy_house.png)

Now we want to see the businesses that are listed at each of the locations we've found. We can combine both correspondance and registered addresses: 

In [6]:
addresses = list(set(correspondance_addresses + registered_addresses))

In [7]:
companies = get_companies(addresses)

Index: 0
3rd Floor, 207, Regent Street, London, England, W1B 3HH
4685 companies registered or corresponding here, 1443 are active.
321 currently active companies registered in 2021
292 currently active companies registered in 2022
216 currently active companies registered in 2020
2211 companies dissolved between years 2-1
464 companies dissolved between years 3-2
174 companies dissolved between years 1-0

Index: 1
C/O BDO LLP, 5 Temple Square Temple Street, Liverpool, L2 5RH
599 companies registered or corresponding here, 2 are active.
2 currently active companies registered in 1999
9 companies dissolved between years 12-11
8 companies dissolved between years 10-9
8 companies dissolved between years 20-19

Index: 2
8 Segedunum Business Centre, Station Road, Wallsend, United Kingdom, NE28 6HQ
189 companies registered or corresponding here, 138 are active.
31 currently active companies registered in 2021
19 currently active companies registered in 2019
16 currently active companies regis

One location that stands out by an absolute mile is "3rd Floor, 207, Regent Street", which a quick Google Search reveals is a "virtual office" run by a company called [Hold Everything](https://www.hold-everything.com/) where businesses can set as a their correspondance address/registered address for £24 a month. They basically just forward your mail on so you can say you have an office on Regent Street. There's even a single board room you can book:

![title](assets/hold.png)

The address gives the impression of legitimacy and exclusivity:

![title](assets/exclusive.png)

Although there are many customers of businesses registered at 207 Regent Street who have experienced scams or poor customer service and don't acknowledge the address is only a mailbox and board room: 

![title](assets/review.png)

Last year the [Daily Mail](https://www.hold-everything.com/) reported on the address and the prevalence of scam businesses registered there: 

![title](assets/dm.png)

Now lets get all the officers that operate at this address by including companies and also a list of the indexes for the companies you'd like to investigate, in this case 207 Regent Street would be index=0 so [0]:

In [ ]:
businesses = get_officers(companies,[0])

It might take a while to run this search on '207 Regent Street' but once we do, we should find some interesting names coming up in our most prolific searchers list (attached full output from a previous run at the bottom of page):

Stuart POPPLETON

- Prolific scammer
- Included in [paradise papers](https://offshoreleaks.icij.org/nodes/56044558)
- Linked to Bulgarian energy oligarch currently seeking 110 millions euros from Serbias government
- TradeInvest90 - [crypto scam](https://www.forexpeacearmy.com/community/threads/scammed-by-tradeinvest90.53201/)
- Also known as Neil Malcolm Stuart John - associated with printing scams for companies such as [UK Print Ltd](http://planettrevormontague.blogspot.com/2016/12/beware-clair-rosina-hunnisett-and-neil.html), [Printed Books Ltd, Offset Printers Ltd](https://www.printweek.com/news/article/print-farming-companies-struck-off)
- Alongside partner been been referred to as the ['Bonnie and Clyde'](https://www.printmonthly.co.uk/News/Business/5792/exposed-unethical-print) of the print industry 
- Google Streetview photograph below from current location where many companies are registered. Can see someone peaking out the window :D

![title](assets/euro.png)

HELVE TCS LIMITED

- Julian Richard BEARDSLEY (director) was found guilty of fraud and money related offences and sentenced to four and half years’ imprisonment at Southwark Crown Court and was disqualified from being a Company Director for 6 years. Daventry fraudster's part in £2.1 million investment and diamond scams which [robbed elderly and vulnerable victims' life savings](https://www.northamptonchron.co.uk/news/crime/fraudsters-part-in-ps21m-scams-robbing-elderly-victims-life-savings-3591064).

![title](assets/diamond.png)

Full output of most prolific business owners operating from 207 Regent Street:

{'CT SECRETARIES LIMITED': 182,
 'POPPLETON, Stuart Ralph': 68,
 'POPPLETON, Stuart': 53,
 'CT SECRETARIES LTD.': 46,
 'HELVE TCS LIMITED': 35,
 'ARORA, Josh Ajit Singh': 32,
 'COOPER, Sarah': 28,
 'CORPORATE AFFAIRS HANDLING LIMITED': 22,
 'SLEEK TECHNOLOGY LTD': 18,
 'TEMPLE SECRETARIES LIMITED': 16,
 'KAPOOR, Rajinder Pal Singh': 14,
 'HARTMAN, Alexander James': 14,
 'SACHS, Francis David': 12,
 'HUGHES, Christopher Paul': 11,
 'COMPANY DIRECTORS LIMITED': 11,
 'KOONES, Charles Crowley': 11,
 'MILGROM, Marc David': 11,
 'APPLETON SECRETARIES LIMITED': 10,
 'HODGKINSON, Paul Roger Dudley': 10,
 'INCORPORATE SECRETARIAT LIMITED': 10,
 'GATCH, Eric Ian': 10,
 'NEDELJKOVIC, Milan': 10,
 'SEWARDS, David': 8,
 'PHILLIPS, Justin Myles Jonathan': 8,
 'MILLWARD INVESTMENTS LIMITED': 8,
 'HICKSON, Katherine Anne': 8,
 'GOLDFINCH, Paul Anthony James': 8,
 'OUDDIR, Abderrahim': 8,
 '7SIDE SECRETARIAL LIMITED': 7,
 'PETRE-MEARS, Sarah Louise': 7,
 'LEBE, Russell Simon': 7,
 'LIAW, Shelly': 7,
 'CLUBE, Laurence Murray': 6,
 'HERN, Jeanette': 6,
 'KLEINMAN, Darrin Andrew': 6,
 'GROUPWISE LIMITED': 6,
 'SCARF, Peter Michael': 6,
 'ATIENO, Taiwo Leo': 6,
 'PANDIA, Sumit': 6,
 'CORNHILL SECRETARIES LIMITED': 5,
 'CUNNINGHAM, Alastair Matthew': 5,
 'HOLLANDER, Colin Ivan': 5,
 'ARORA, Harmeena': 5,
 'FORM 10 SECRETARIES FD LTD': 5,
 'FORM 10 DIRECTORS FD LTD': 5,
 'GERARD, Bruno Andre, Joseph, Romain': 5,
 'BOURGEOT, Thierry Francois': 5,
 'SABIRY, Chafik': 5,
 'MARJANOVIC, Tomislav': 5,
 'SAUNDERS, Ashton': 5,
 'LANDAU, Elizabeth Heather': 5,
 'CHAPMAN, Craig': 5,
 'CHAPMAN, Craig John': 5,
 'JONES, Damien': 5,
 'NEWTON, Jamie': 5,
 'MORANO, Francois': 5,
 'ZAJOTTI, Francesco': 5,
 'BRUCE-MORGAN, Torsten Morris Tomas': 4,
 'MEADLEY, Nicola Joy': 4,
 'WILDERSPIN, Mark Paul': 4,
 'COOKE-PRIEST, James Robin Bruce': 4,
 'PEMEX SERVICES LIMITED': 4,
 'TARCHINI, Romain Joseph Michel': 4,
 'TULI, Suneet Singh': 4,
 'ALLAN, Angela Joyce': 4,
 'CRYSTAL COMMERCE SA': 4,
 'GUERNSEY CORPORATE SECRETARIES LIMITED': 4,
 'AMERIKANOS, Georgios': 4,
 'ZALES, Samuel': 4,
 'DAS, Utpal Kumar': 4,
 'LS BUSINESS SERVICES LIMITED': 4,
 'WARSHAW, Nicholas Anthony Beresford': 4,
 'SINGH, Kunwar Ajit': 4,
 'HOWELL, Rupert Cortlandt Spencer': 4,
 'MACHRAY, Philip James': 4,
 'TELLIEZ, Daniel Paul': 4,
 'BURKE, Daniel James': 4,
 'TAN, Colin': 4,
 'WEST, Timothy': 4,
 'MITCHELL, Mark': 4,
 'HUNTSMOOR LIMITED': 4,
 'HUNTSMOOR NOMINEES LIMITED': 4,
 'HCS SECRETARIAL LIMITED': 4,
 'PHILLIPS, Patricia Susan': 4,
 'NOMINEE SECRETARY LTD': 4,
 'PORTESS, Thor Hamilton Elick': 4,
 'BRANSON, Thomas Andy': 4,
 'STAYTON, Robert James Leslie': 4,
 'CUBITT, Sean': 4,
 'ARIZA-SANCTUARY, Rafael': 4,
 'BASSI, Manisha': 4,
 'COLE, Andrew': 4,
 'MALIK, Nadeem': 4,
 'CARRUTHERS, Matt': 4,
 'MISTRY, Bianca': 4,
 'WALLIS, Frances Jane': 4,
 'AL FURAIH, Naser, Dr': 4,
 "O'CONNOR, Nicholas": 4,
 'CHISHOLM, Oreial': 4,
 'EMANUEL, Jerome': 4,
 'JANES, James': 4,
 'MARRI, Vijayanjan': 4,
 'MCINTOSH, Barrington': 4,
 'MORETON, Thomas': 4,
 'SEYMOUR, Gary Robert': 4,
 'VELAUTHAM, Kirikaran': 4,
 'WAI, King': 4,
 'WARRILOW, Thomas': 4,
 'CAPPE, Bertrand William': 4,
 'HERNANDES, Bernard Patrice': 4,
 'KAUR DOAL, Anjali': 4,
 'BOSOMWORTH, Danyl Lee': 3,
 'MARSON, Richard John': 3,
 'REEVE, Damon Andrew': 3,
 'RAHAMAN, Alexander': 3,
 'SWITZER, James Scott': 3,
 'HARVEY, Jayne Denise': 3,
 'HOMERIC LIMITED': 3,
 'CORNHILL REGISTRARS LIMITED': 3,
 'TERTHUR TRADING LIMITED': 3,
 'KEYSTONE INVESTMENTS LIMITED': 3,
 'APPLETON DIRECTORS LIMITED': 3,
 'AMOOTI, Kyamuwendo Elijah': 3,
 'GOLDFINCH, Scott James': 3,
 'TMP BUSINESS SERVICES LTD': 3,
 'MATHARU, Jaswinder Singh': 3,
 'ARORA, Chatter Singh': 3,
 'CALLAGHAN, Emma': 3,
 'HANOVER DIRECTORS LIMITED': 3,
 'SENATOR CAPITAL GROUP LLC': 3,
 'PASTERNAK, Robert Daniel': 3,
 'WILKINSON, Robert Ronald': 3,
 'DUFFY, James William': 3,
 'TECHNOLOGY LTD, Sleek': 3,
 'CARLSE, Ivan Trevor': 3,
 'MUSONDA, Rabecca Lubaya': 3,
 'GOODRICH, James': 3,
 'PASARIC, Marin': 3,
 'COM, Olivier': 3,
 'KITCHEN, Emma': 3,
 'KAHAN, Barbara': 3,
 'LONDON LAW SECRETARIAL LIMITED': 3,
 'RANDHAWA, Varinder Singh': 3,
 'GOUTARD, Anthony Joel': 3,
 'SPADA, Maria Cristina': 3,
 'SACHDEV, Byron Malik': 3,
 'BOAGE, Ian Neal': 3,
 'JEWELL, Malcolm Alec': 3,
 'JEVTOVIC, Zoran': 3,
 'CHESNEAU, Romann Franck': 3,
 'NOURI, Mourad Abderrahmane Maurice': 3,
 'THOMPSON, Bruce': 3,
 'SYMES, Darren': 3,
 'BARRASS-BROWN, Wayne Hunter': 3,
 'COLE, Jennifer': 3,
 'COTTERILL, Leigh Justin': 3,
 'TAYLOR, Dean': 3,
 'DAI, Jianfeng': 3,
 'ABREO, John': 3,
 'MANTEY, Guy-Michel': 3,
 'REID, Joash': 3,
 'OWARE, Chanel': 3,
 'PREEN, Melissa': 3,
 'SHAHI, Abhishek, Dr': 3,
 'SMITH, Andrew Raymond Marc': 3,
 'WALKER, Benjamin Alexander': 3,
 'WECKER, Jean-Marc Guy': 3,
 'DA SILVA COUTINHO, Konrad': 3,
 'HARISH, Casey': 3,
 'AHMAD, Adeel': 3,
 'AHMAD, Sajjad': 3,
 'ALAJMI, Khaled': 3,
 'ALMUTAIRI, Fahad': 3,
 'ARTIS, Graham': 3,
 'ATIENO, Taiwo': 3,
 'BALAYAN, Levon': 3,
 'BEVIS, Burint': 3,
 'BRESSLING, Andre': 3,
 'CHASTANET, Franck': 3,
 'HUDSON, Philip Lawrence Jack': 3,
 'CLAY, Simon Marc': 3,
 'GROSSMAN, Maxwell Aaron': 3,
 'COX, Geoffrey Allen': 3,
 'DIXON, Paul': 3,
 'MILLS, Michael John': 3,
 'JOHNS, Richard': 3,
 'MORRIS, Steve': 3,
 'ROBINSON, Mark': 3,
 'RAJA, Susie': 3,
 'RAWLINSON, Colin': 3,
 'SANGHA, Jasvinder': 3,
 'THOMAS, David Benjamin': 3,
 'TILLEY, Eathan': 3,
 'GURAZIU, Rudi': 3,
 'HAQUE, Mohammad': 3,
 'JAMES, Jonathon': 3,
 'CHALMERS, Andrew Craig': 2,
 'JACKSON, Nicholas': 2,
 'MCGRATH, Don Geoffrey': 2,
 'RAINEY, Samuel': 2,
 'WARD, Michael Joseph': 2,
 'METZGER, Joshua': 2,
 'CADOGAN, Timothy': 2,
 'GENTRY, John': 2,
 'GOMBOS, Rick': 2,
 'MCPHERSON, Douglas': 2,
 'SDG SECRETARIES LIMITED': 2,
 'BILEFIELD, James Daniel': 2,
 'IBRAHIMI, Khalil Ahmad': 2,
 'SDG REGISTRARS LIMITED': 2,
 'CHIMA, Parmeet Singh': 2,
 'NICHOLLS, William Richard Hardy': 2,
 'COMBINED SECRETARIAL SERVICES LIMITED': 2,
 'COOKE-PRIEST, James Robin': 2,
 'SOAR, Phil William': 2,
 'BLAIR, Charlotte Miranda Hacker': 2,
 'AGGARWAL, Nina': 2,
 'SOAMES, Christopher Paul': 2,
 'MERIDIAN SECRETARIES LTD': 2,
 'MERIDIAN PARTNERS LTD': 2,
 'CURRAN, Anthony': 2,
 'ALBANY MANAGERS LIMITED': 2,
 'ELSTOW INVESTMENTS LIMITED': 2,
 'COLCLOUGH, James Martin Joseph': 2,
 'SWIFT INCORPORATIONS LIMITED': 2,
 'INSTANT COMPANIES LIMITED': 2,
 'PERRET, Sylvie, Mme': 2,
 'STRUB, Wilhelm Irving': 2,
 'AMERSHAM SERVICES LIMITED': 2,
 'HALCO SECRETARIES LIMITED': 2,
 'TULI, Raja Singh': 2,
 'PETRIE, Rebecca': 2,
 'FIDUCIARY DIRECTORS LIMITED': 2,
 'EVANS, Glyn': 2,
 'BROWN, Michael': 2,
 'TILLEY, Jason': 2,
 'CHALFEN SECRETARIES LIMITED': 2,
 'CHALFEN NOMINEES LIMITED': 2,
 'ZAMORA, Javier Esquivel': 2,
 'ELDRIDGE, Andrea Lee': 2,
 'SYMONDS, Amanda Jane': 2,
 'FREDO, Scot': 2,
 'PATTON, Kathleen': 2,
 'KENNY, Kristen Victoria': 2,
 'TREVISAN, Jason Michael': 2,
 'MURASA, Rose': 2,
 'VANDEVELDE, Wim': 2,
 'SMITH, Andrew Donald': 2,
 'CHILDS, Maureen Anne': 2,
 'PRICE, Simon David': 2,
 'HEAP, Andrew Gordon': 2,
 'DELIA, Colin': 2,
 'THUILLARD, Jan': 2,
 'KUMAR, Manoj': 2,
 'MAVANI, Neeta Avinash': 2,
 'MILLERS ASSOCIATES LIMITED': 2,
 'ADEX, Samson Tosin': 2,
 'OJUOLAPE, James Ayo': 2,
 'STEANNE INDUSTRIES (OVERSEAS) LIMITED': 2,
 'HOLDEN, Daniel James': 2,
 'HOUGUEZ-SIMMONS, Jacqueline': 2,
 'MUSONDA, Chama Chilufya': 2,
 'MILLER, Andrew Michael': 2,
 'PHILLIPS, Louise Ann': 2,
 'OXLEY, Claire Ann Marie': 2,
 'BOURNE-TURNER, Kenneth Ian': 2,
 'COTTRELL, Gary Bernard': 2,
 'TURNER, Karen Louise': 2,
 'WILSON, Ray': 2,
 'CAPITAL SECURITIES LIMITED': 2,
 'FERRIS, Julie Ann': 2,
 "O'GRADY, John Francis": 2,
 'BROADOAK DIRECTORS LIMITED': 2,
 'HOGG, Ian Christopher': 2,
 'PHILLIPS, Clive Anthony': 2,
 'BALAN, Floriana': 2,
 'HAMACHE, Mustapha': 2,
 'NOMINEE DIRECTOR LTD': 2,
 'HONOREM EXPERTISE SARL': 2,
 'HUNT, John Ferry Shackleton': 2,
 'SILVESTRI, Fabio': 2,
 'OWUSU-ANSAH, Joseph': 2,
 'MILL, George': 2,
 'FASSI FEHRI, Zeyd': 2,
 'MCMURRAY, Colin Andrew': 2,
 'UZOKWE, Izunna, Dr': 2,
 'AINSWORTH, Robert': 2,
 'HURWORTH, Aderyn': 2,
 'GASCOIGNE PEES, Robert': 2,
 'WILLIAMS, Mark Gillin': 2,
 'HABER, Jonathan': 2,
 'HOWLETT, Stephen': 2,
 'DUPORT SECRETARY LIMITED': 2,
 'DUPORT DIRECTOR LIMITED': 2,
 'TONEV, Boyan': 2,
 'RACHEV, Hristo Valentinov': 2,
 'KAPOOR, Pavan': 2,
 'KAPOOR, Raj': 2,
 'VAID, Lajdeep': 2,
 'KHANNA, Punita': 2,
 'KHANNA, Anshuraj': 2,
 'LOUW, Mark': 2,
 'MILLAR, Ryan Cary': 2,
 'WILSON, Blaine Robert': 2,
 'PURCELL, Frank': 2,
 'WEISS, Roland Jean-Louis': 2,
 'BONIS MARCHANDIN, Claire': 2,
 'WICKENS, Martin John': 2,
 'ZATLAND, Kate': 2,
 'WATKINS, John': 2,
 'AKPINAR, Mehmet': 2,
 'DUBRO, Anna': 2,
 'BUJDO, Marinko': 2,
 'NAYYAR, Angela': 2,
 'COMET, Nicolas Bernard': 2,
 'PIGNOTTI, Alessandro': 2,
 'SCARF, Peter Michael Edward': 2,
 'STAYTON, Jennifer Mary': 2,
 'HUXTABLE, Craig': 2,
 'JONES, Mark Anthony': 2,
 'GRIFFITHS, Ernest Thomas': 2,
 'JURNAIL, Gurnam Singh': 2,
 'MARTIN, Henry': 2,
 'LEWIS, Sarah Jane': 2,
 'LOTAY, Tarandeep': 2,
 'LOTAY, Mohinder Singh': 2,
 'AVLANI, Dilip': 2,
 'SODHA, Mahavir Kumar': 2,
 'LUMINGO, Messias Messias': 2,
 'SHEKONI, Oladapo Abdul-Azeez': 2,
 'TRUTSCH, Philipp': 2,
 'MATTHEWS, Ian Ronald': 2,
 'MATTHEWS, Samuel Eric': 2,
 'MONTGOMERY, Richard': 2,
 'PATAIA, Vittorio': 2,
 'MCCALLUM, David': 2,
 'MCGARVEY, Paul': 2,
 'MORRIS, Paul': 2,
 'LONDON LAW SERVICES LIMITED': 2,
 'GUELLARD, Sophie Jane': 2,
 'RANDHAWA, Balbir Kaur': 2,
 'RANDHAWA, Lakhwant Singh': 2,
 'RIGBY, Alex': 2,
 'PANCHAL, Kiran Illais': 2,
 'SHAH, Raza Ali': 2,
 'TEWOLDE, Sirac': 2,
 'TEWOLDE, Senait': 2,
 'VINCENT, Christopher': 2,
 'VALAITIS, Peter Anthony': 2,
 'ZAHOUA, Zahra': 2,
 'ABDUL AZIZ ABDUL RASHEED, Kinza Tania': 2,
 'ADEM, Metin': 2,
 'KASIM, Mustafa': 2,
 'AMENONYOH, Tatah': 2,
 'HANIF, Sara': 2,
 'COOPER, Richard Laurence': 2,
 'ASGARI, Soheil, Dr': 2,
 'BAILEY, Chloe': 2,
 'BELYAEV, Maxim': 2,
 'BERATUNGSZENTRUM FUER TUERKISCHE UNTERNEHMEN LTD.': 2,
 'BONNINGUES, Franck Georges': 2,
 'BOULANGER, Stephane': 2,
 'MUTALE, Justina': 2,
 'DUSABIMANA, Honore, Dr': 2,
 'BROWN, Peter': 2,
 'CAMPBELL, Lynda': 2,
 'CARPENTIER, Eric Christian': 2,
 'DAVID, Bastien Francois Alain': 2,
 'CHRETIEN, Francois Charles Paul': 2,
 'FLOREA, Costel Cristian': 2,
 'COX, Jemma': 2,
 'CUILLERON, Martial Lionel': 2,
 'WONG, Jowyn': 2,
 'MURPHY, Brian Christopher': 2,
 'WILLIAMS, Richard Keith': 2,
 'EL-RABAA, Rinad': 2,
 'FARAH, Mohamed Issa': 2,
 'FLAHAUT, Michael': 2,
 'GOLDSMITH, Daniel John': 2,
 'KUREK, Marcin': 2,
 'GOUVEIA, Jose': 2,
 'GREENWOOD, Vineeta': 2,
 'KHOURY, Ziad': 2,
 'HALL, Lauren': 2,
 'HOLT SNR, John': 2,
 'INTERTRUST (UK) LIMITED': 2,
 'MEHTA, Ravin Harshad': 2,
 'VIANA CABRAL NETTO, Olavo': 2,
 'ISHMAEL, Aadam': 2,
 'JOHNSON, Aldene Mary': 2,
 'NOVAK, Jaroslav': 2,
 'NUSSBAUM, Alan Brian': 2,
 'JOHAL, Ramandeep Singh': 2,
 'KENDRICK, Leight': 2,
 'KHAN, Humma Mazhar': 2,
 'ANAND, Shaila': 2,
 'LEADER, Marc': 2,
 'MANTLE, Myles Justin Bruce': 2,
 'LOAN, Mark': 2,
 'LOFTHOUSE, Louis Piers': 2,
 'MANSOOR, Muhammad': 2,
 'MAYO, Douglas James': 2,
 'COLMAN, Christopher David': 2,
 'LAWRENCE, Andrew James': 2,
 'PODMORE, Mark Winston': 2,
 'SALIB, Hany Wadea Elshahat': 2,
 'MINKIEWICZ, Anne Sophie': 2,
 'NAPOLITANO, Letizia': 2,
 'MAHMOOD, Zahid': 2,
 'OMOVBUDE, Omoneka Osa': 2,
 'OPTIMA SECRETARIES LIMITED': 2,
 'OWEN, Gareth Wyn': 2,
 'POLIAKOV, Alexei': 2,
 'UBRIG, Alain Jean': 2,
 'VAILLANT, Bertrand Marie': 2,
 'ZIEGLER, Stephane Marie': 2,
 'PORCU, Alessandro': 2,
 'IANIERO, Leonardo': 2,
 'PRAF, Alin Nicolae': 2,
 'PROGEN SOLAR': 2,
 'RICHARD, Etienne Pierre, Michel': 2,
 'ROBERT, Christophe': 2,
 'ROBERTS, Vivienne Sue': 2,
 'SABADO, Melanie': 2,
 'SINGCUENCO, Katrina': 2,
 'SEGEL, Adam': 2,
 'SIMON, Wayne': 2,
 'ST.HILL, Kathleen': 2,
 'STAN, Madalin': 2,
 'TAS, Mehmet': 2,
 'GUNDOGDU, Omer': 2,
 'THOMPSON, Julian': 2,
 'VANHINSBERGH, Kevin': 2,
 'WELLS, Ruxandra Luana': 2,
 'WELLS, Richard Hase': 2,
 'FUTURE II LTD': 2,
 'ABDULLAHI MUSA HAI, Ahmed': 2,
 'ADATSI, Franklin Jerry Kojo': 2,
 'FERNANDEZ-PRADOS, Elena Maria': 2,
 'AGUTTER, Emily': 2,
 'AHMADI, Hossein': 2,
 'AJAYI, David': 2,
 'AJAYI-OBE, Olawale Olakunle': 2,
 'AKINLUYI, Tolulope Stephen': 2,
 'ALLEN, Martin Andrew': 2,
 'ARCURI, Jennifer': 2,
 'HICKEY, Matthew James': 2,
 'WESTCO DIRECTORS LTD': 2,
 'AYE, Emily': 2,
 'BADIANE, Bakary': 2,
 'BAKER, Phillipa': 2,
 'BARKER-GOLIDE, Christine': 2,
 'BARRASS-BROWN, Wayne': 2,
 'BARROWCLIFFE, Maxine': 2,
 'BENBOW, Shaun, Dr': 2,
 'BLANCO, Sergi Mora': 2,
 'LOPEZ, Manuel Sanchez': 2,
 'POYATOS, Rafael Nevado': 2,
 'BLISS, James': 2,
 'BOUMAZOUNA, Leila': 2,
 'BOURNE, Patricia': 2,
 'REES, Nicholas': 2,
 'BROU, Bertin': 2,
 'NOAR BROWNE': 2,
 'BURLACU, Daniel': 2,
 'CARTER, Kaya': 2,
 'QUARRIE, John Elliott': 2,
 'CHANDRASEKERAN, Udeshika': 2,
 'CHEUNG, Hilary Hin': 2,
 'CHIKAMBA, Priscilla': 2,
 'CHIRINDA, Frank': 2,
 'CLARK, Eugene Anthony': 2,
 'COATES, Faye': 2,
 'COLEMAN, Tony': 2,
 'CONNIS, Jadon': 2,
 'COWLING, William': 2,
 'CRITCHLOW, Fabian': 2,
 'YOUNG, Jay Arron': 2,
 'BARTON, Elliot Henry': 2,
 'CONNOLLY, Kieran Thomas': 2,
 'GRIFFITHS, Paul Anthony': 2,
 'MONTEIRO DE FREITAS, Angela De Fatima': 2,
 'DALY, Veronica': 2,
 'DANQUAH, Jason': 2,
 'DAVIES, Angela Chipo Ulemu, Dr': 2,
 'DENCIC, Aco': 2,
 'LASS, Rosemary Anne': 2,
 'RANDLE, Vivien Edwina': 2,
 'DOLAN, Edward': 2,
 'DOSANJH, Ardeep': 2,
 'EFFANGA-PETER, Eno Jasmine': 2,
 'FALADE, Moranti': 2,
 'FEJZOLLI, Amarildo': 2,
 'FERRAZ, Goncalo Nuno Andrade': 2,
 'JEWELL, Matthew Malcolm': 2,
 'FINNIGAN, Diana': 2,
 'FISHER, Greig': 2,
 'FORRESTER, Delvon': 2,
 'GAVIN, John Thomas': 2,
 'GEAREY, Jacqueline Margaret': 2,
 'GEORGE JUSTIN, Alwyn Clittus': 2,
 'GITTOES-BREE, Stephanie Elaine': 2,
 'GLADWIN, Yvone': 2,
 'SCOTT, Jeff': 2,
 'GRANGIER, Arnaud Pascal': 2,
 'GROOM, Craig Michael': 2,
 'GURLU, Vedat': 2,
 'HALL, Suzie': 2,
 'HAM, David': 2,
 'HASSAN, Md': 2,
 'HATHAWAY, Beth': 2,
 'HEARN, Anthony': 2,
 'HEW, Kok-Hoong': 2,
 'HOLMES, Glenn': 2,
 'HOPE, Sandra': 2,
 'HORSWILL, Jill': 2,
 'IDRISSI, Abdulmounime': 2,
 'IKEJIANI, Ezinne': 2,
 'INAYATHULLA, Mohamed': 2,
 'PAL, Sumit': 2,
 'IRIMIA, Robert': 2,
 'JEFFERSON, Alison Jane': 2,
 'JOHN, Ceri Richard': 2,
 'JOHNS, Beverley': 2,
 'JUBA, Vidal Adedapo': 2,
 'SOUDANI, Abdelhamid': 2,
 'MEHMI, Reeni Kumari': 2,
 'KAMMAR, Rashmi': 2,
 'KEEFE, Karen': 2,
 'KELLY, John': 2,
 'KENNEDY, John': 2,
 'KILFOY, Anthony': 2,
 'KING, Rebecca': 2,
 'KLUK, Linda': 2,
 'KUMAR, Anjana': 2,
 'KUPRYS, Mantas': 2,
 'PIKALOV, Dmitrii': 2,
 'LAWRENCE, Matthew Charles': 2,
 'TAKIEDDINE, Karim Markus': 2,
 'LEDDER, Philip': 2,
 'ROBINSON, Jonathan Charles': 2,
 'LEMKECHER, Majed': 2,
 'LI, Xiao Chun': 2,
 'LIGHTFOOT, Steve': 2,
 'ANGILELLA, Marco': 2,
 'DIJKSTRA, Antje Rona': 2,
 'LAWSON, Nicholas St John': 2,
 'MACAULEY, Olayinka': 2,
 'MANDORA, Bhavesh': 2,
 'MANEA, Ion': 2,
 'MARGOLIN, Ryan': 2,
 'MARIN, Glenn': 2,
 'MCMAHON, Paul': 2,
 'RICHARDS, Greg': 2,
 'MARTIN, John': 2,
 'MARTONFI, Alexandru Robert': 2,
 'MARTYN, Elizabeth': 2,
 'MATIVI, Farai Alfred': 2,
 'MCAULIFFE, Adam Michael': 2,
 'GORDON, Jamie': 2,
 'MCKEVITT, Brian': 2,
 'JONES, Nick': 2,
 'MEDINA, Jose': 2,
 'BERARDO, Angela Teresa': 2,
 'WARDLE, Charlotte Victoria': 2,
 'MOKWELE, Mungombe': 2,
 "O'SHAUGHNESSY, Myles": 2,
 'PICOTTE, Christopher James': 2,
 'MUGHAL, Asim': 2,
 'MURRAY, Owen': 2,
 'NADARAJAH, Anju': 2,
 'NASTASIA, Anca': 2,
 'NATANYAN, Arthur': 2,
 'NICHOLSON, Kevin': 2,
 'BANKS, Shay Ifeanyi': 2,
 'NORTH, Barbara': 2,
 'OBASEKI, Wickcliffe': 2,
 'ODOKAMA, Joyce': 2,
 'OKUNGBOWA, Irene': 2,
 'OLORUNLERO, Opeyemi': 2,
 'ORLANDINI, Guido': 2,
 'OUMOUMEN, Hamid': 2,
 'PEIKOS, Phillip': 2,
 'PIRCALABU, Elena': 2,
 'POLIARUSH, Vitalii': 2,
 'POLONSKI, Vyacheslav': 2,
 'PRICE, Robin': 2,
 'PRIDDEN, Joanne': 2,
 'RABIA, Dionis': 2,
 'RAYMOND, Moshe': 2,
 'REDDY, Martin': 2,
 'ROBERT, Vashisth': 2,
 'ROBINSON, Tony': 2,
 'ROSE, Steven': 2,
 'NANJI, Asani Ashvin': 2,
 'ROUSOPOULOS, Georgios': 2,
 'SAHDEV, Krishan': 2,
 'SAMUEL, David': 2,
 'SAVAGE, Sarah': 2,
 'MADDALUNI, Giovanni': 2,
 'SEXTON, Timothy': 2,
 'SLEE, Richard Peter': 2,
 'SMITH, Pascaline': 2,
 'SPEK, Patricia': 2,
 'STIUBEIANU, Carmen Marioara': 2,
 'SABBATINI STEWART, Roberta': 2,
 'BANDANA LTD': 2,
 'THERON, Theodorus': 2,
 'TANG, Chi Sim': 2,
 'THOMAS, Winston': 2,
 'AKDERE, Süleyman': 2,
 'UGHO, Chioma': 2,
 'UNIGWE, Rachel': 2,
 'VARLAN, Iulian': 2,
 'WALLIS, David John': 2,
 'WARD, Eleanor Rosamund': 2,
 'WARWICK, Lindsey Jean': 2,
 'WASHINGTON, Dayo': 2,
 'WENDT, Lars': 2,
 'WHITE, Farah Jane': 2,
 'WIJNHARD, Clyde Carlo': 2,
 'WILKS, Mark': 2,
 'WILLIAMS, Sarah': 2,
 'WOODS, Edward': 2,
 'YANG, Wenli': 2,
 'ABDULAZIZ, Yousif': 2,
 'ABDULSALAM, Nabil': 2,
 'ABRAHAM, Jaime Ilolo': 2,
 'ABULQASIM, Ismail': 2,
 'ADENEKAN, Olujimi': 2,
 'ADEWOLE, Oluwadamilare': 2,
 'ADJEI, Samuel': 2,
 'ADUAKA, Christy Ify': 2,
 'REHMAN, Danish': 2,
 'AHMED EBRAHEM ALMASOUD, Mohammad': 2,
 'AHMED, Ripon': 2,
 'AHMED, Rubina': 2,
 'AKUCHUKWU, Smart': 2,
 'AL AZAWI, Ilham Rashid': 2,
 'AL ENEZI, Jamal': 2,
 'AL ENEZY, Olia': 2,
 'AL ENIZI, Iman': 2,
 'AL-NAJJAR, Musaeb Mohammed': 2,
 'AL-ZYADI, Salwan': 2,
 'ALAJMI, Abdullah': 2,
 'ALAJMI, Fahad': 2,
 'ALAJMI, Hanan': 2,
 'ALAJMI, Maha': 2,
 'ALAJMI, Mashael': 2,
 'ALAJMI, Meshal': 2,
 'ALAJMI, Saad': 2,
 'ALAJMY, Aly': 2,
 'ALALI, Humoud': 2,
 'ALAMEER, Hameed': 2,
 'ALANZI, Abdulrahman': 2,
 'ALASOW, Sahra Abdullahi': 2,
 'ALAWADHI, Abdulaziz': 2,
 'ALAWADI, Abdulla, Dr': 2,
 'ALAWADI, Khaleel, Dr': 2,
 'ALAZMA, Majed': 2,
 'ALDHUFAIRI, Mohammad': 2,
 'ALDOUSARI, Mubarak': 2,
 'ALDUWAISAN, Muna': 2,
 'ALENEZI, Amany': 2,
 'ALENEZI, Ameenah': 2,
 'ALENEZI, Dalal': 2,
 'ALENEZI, Fatima': 2,
 'ALENEZI, Manal': 2,
 'ALENEZI, Nawal': 2,
 'ALENEZI, Yousef': 2,
 'ALHARBI, Bdour': 2,
 'ALHASSAN, Abdullah': 2,
 'ALHINDI, Mona': 2,
 'ALHINDI, Najat': 2,
 'ALHUNAIF, Emad': 2,
 'ALI, Miriam': 2,
 'ABBAS, Ali': 2,
 'ALIYEV, Farhad': 2,
 'ALJASSAS, Nadyah': 2,
 'ALKANDARI, Waleed': 2,
 'ALKANDERI, Sohaib': 2,
 'ALKHALEFAH, Dawood': 2,
 'ALMARAIKHI, Suad': 2,
 'ALMAZER, Salman': 2,
 'ALMERAIKHI, Ghanimah': 2,
 'ALMUDHAYAN, Ahmad': 2,
 'ALMUDHAYAN, Hamed': 2,
 'ALMUTAIRI, Hanan': 2,
 'ALMUTAIRI, Meshal': 2,
 'ALMUTAIRI, Mohammed': 2,
 'ALQATTAN, Qasem': 2,
 'ALRABAH, Yousef': 2,
 'ALRABEIAH, Fetouh': 2,
 'ALRABIAH, Aminah': 2,
 'ALRASHED, Husain': 2,
 'ALRASHIDI, Wasemeya': 2,
 'ALSANAD, Saja': 2,
 'ALSARDI, Shatha': 2,
 'ALSHAIJI, Ali': 2,
 'ALSHALAHI, Abdulrahman': 2,
 'ALSHAMMAA, Majed': 2,
 'ALSHATTI, Yaqoub, Dr': 2,
 'ALSHEMALI, Abdulrahim': 2,
 'ALSUBAIE, Naji Fahad': 2,
 'ALTAWHEED, Elham': 2,
 'ALYAHYA, Ahmad': 2,
 'ALYAQOUB, Amina': 2,
 'ALYAQOUB, Khaledah': 2,
 'ALYAQOUB, Naheda': 2,
 'ALYAQOUT, Wafa': 2,
 'ALZAID, Ibrahim': 2,
 'ALZEED, Fahad': 2,
 'AMANKWAA, Cosmos': 2,
 'AMJAD, Muhammad': 2,
 'ANDERSON, Jemma': 2,
 'ANDREEV, Nikolay': 2,
 'ANDREEVA, Elena': 2,
 'ANIEBONAM, Uche Snr.': 2,
 'ANNAN, Kenneth, Dr': 2,
 'ANTKOWIAK, Aneta': 2,
 'ARNELL, Katherine Ann': 2,
 'ASCOTT, Marcus': 2,
 'ASHCROFT, Frank': 2,
 'ASHTON, Michael': 2,
 'SECRETARIAT OFFICERS LIMITED': 2,
 'AYRES, Richard Desmond': 2,
 'AZIZ, Tariq': 2,
 'TRUTA, Lavinia': 2,
 'BAARS, Raymond': 2,
 'BACHAS, Paul': 2,
 'BAGLEY, Mark': 2,
 'BAILEY, Jamie': 2,
 'BAKHTADZE, Nina': 2,
 'BALALA, Maha-Hanaan, Dr': 2,
 'BALANDIN, Sergey': 2,
 'BALDWIN, Keith': 2,
 'BALL, Jonathan': 2,
 'BALTAZZI, Gianluigi': 2,
 'BAMBER, Claire': 2,
 'BANGALORE VENKATARAMU, Chandan': 2,
 'BANJOKO, Adekunle': 2,
 'BANKS, David': 2,
 'BARBARIOLI CUNHA, Raianny': 2,
 'BARBER, Mark': 2,
 'BARRETT, Nicolas': 2,
 'BARTRAM, Richard': 2,
 'BARWICK, Elizabeth': 2,
 'BARZEY, Alexandra': 2,
 'COLA, Vincent Patrice': 2,
 'BAWA, Sanmukh': 2,
 'BAYLISS, Simon': 2,
 'BEAGLES, Edwin': 2,
 'BECKFORD, Carleen Nicole': 2,
 'BEDDING, James': 2,
 'BEGLAN, Peter': 2,
 'BELL, Adam': 2,
 'BELOV, Alexander': 2,
 'BELTJES, Steven Adriaan Johan': 2,
 'BENJAMIN, Michael': 2,
 'BENNETT, Kyle': 2,
 'BERGMAN, Maxine': 2,
 'BERRY, Matthew': 2,
 'BHEEMARASETTY, Chalam Santosh Ram Mohan': 2,
 'SWABY, James': 2,
 'BIRCH, Harvey': 2,
 'BIRLEY, Adam': 2,
 'BLACKABY, Paul': 2,
 'BLISS, Jonathan Douglas': 2,
 'BOLENGE-CARTIERÈ, Fabian': 2,
 'BOLT, Bonnie': 2,
 'BOUKARABILA, Abdellah': 2,
 'BOURLOT, Caireen': 2,
 'BOWIE, Andrew': 2,
 'BRANNAN, Calum': 2,
 'BRITTON, Adam': 2,
 'BROWN, Alexander James': 2,
 'BROWN, Malcolm Peter': 2,
 'BUKVIC, Alan James': 2,
 'BULLEN, Joel': 2,
 'BULU, Tugce': 2,
 'BUNKER, Miles': 2,
 'BURASZ, Artur': 2,
 'BURNS, Richard': 2,
 'LE BAIL, Bruno': 2,
 'CADDICK, Rebecca': 2,
 'CALDERWOOD, Ryan': 2,
 'CALIN, George': 2,
 'CAMERON, James': 2,
 'CAMPBELL, Peter': 2,
 'CAMPBELL, Robert': 2,
 'CANNON, Janice Lesley': 2,
 'CAPEWELL, William': 2,
 'CASH, Daniel': 2,
 'CAWTHORNE, Ben': 2,
 'CHAPMAN, Virginia Wangari': 2,
 'CHAPMAN, William David': 2,
 'CHARLES, David': 2,
 'CHEN, Wen Hao': 2,
 'CHERICHI PURAYIL, Saleema': 2,
 'CHOMUZINDA, Ngoni': 2,
 'CHOWDHURY, Jahanara': 2,
 'CHRISTMAS, James Stephens': 2,
 'CHRISTOU, Stefan Yiannis': 2,
 'THAMNUSARN, Pariya': 2,
 'CLARIDGE, Matthew-James': 2,
 'CLARKE, Hayley': 2,
 'CLARKE, Jade': 2,
 'CLEMENT, Paul': 2,
 'CODJO, Sarah': 2,
 'DONNE, Richard': 2,
 'CONNERTON, Dylan': 2,
 'CONWAY, Mark': 2,
 'COOKE, Julian David': 2,
 'CORDERO, Vanessa': 2,
 'COTTET, Renaud': 2,
 'CRADDOCK, Michael': 2,
 'CRAIG, Darren': 2,
 'ABDURRAHMAN, Umar': 2,
 'ALAMUTU, Adeolu': 2,
 'FAYOMI, Oloruntoba': 2,
 'WAHLE, Fabian Felix, Dr': 2,
 'ZHANG, Hong Zhen': 2,
 'BOETTCHER, Ludmila': 2,
 'LARSEN, Kristian Sveen': 2,
 'CUDWORTH, Philip Stephen': 2,
 'HAYWARD, Philip John': 2,
 'CYD TRENCH, Abigail': 2,
 'DAHAR, Mohsin': 2,
 'DALTON, Mark Andrew': 2,
 'DALVERNY, Celine': 2,
 'DAMIAN, Lica Marcel': 2,
 'DAVEY, Patrick': 2,
 'DAVID, Daniel': 2,
 'DAVIES, Jamie': 2,
 'DE ANDRADE, Marcos': 2,
 'DE MENDONCA, Patrica': 2,
 'DEMIREL, Abdulkadir': 2,
 'DENNENY, Simon': 2,
 'DOBROWOLSKI, Agnieszka': 2,
 'DOGAN, Engin': 2,
 'DOLAN, Max': 2,
 'DORMAN, Ben': 2,
 'DOSSO, Mamadou': 2,
 'DOUGHTY, Lynn': 2,
 'DOUGLAS, Jeffery': 2,
 'LEWIS, Paul David': 2,
 'DRISCOLL, Peter': 2,
 'DUFFY, David William': 2,
 'DUNFORD, Paul': 2,
 'DUNN, Daniel': 2,
 'DWYER, Brent': 2,
 'DYER, David': 2,
 'EBANKS-BLAKE, Sylvan': 2,
 'EBRAHIM, Suad': 2,
 'EDWARDS, Natasha Dawn': 2,
 'EL MARNISSI, Mouhcine': 2,
 'EL-HADDAD, Otman': 2,
 'ELIAS, Mozelle': 2,
 'ELLIOTT, Nicholas': 2,
 'ELLISSON, Pam': 2,
 'ENNIS, Kara Lois': 2,
 'ENWELIM, Miriam': 2,
 'ETE, Malik': 2,
 'ETIENNE, Leona': 2,
 'EUGENE, Cora': 2,
 'EVANS, John Albert Charles': 2,
 'FACEY, Kyle': 2,
 'FAIRCLOUGH, Rupert': 2,
 'FAPOHUNDA, Damilola': 2,
 'FARHANG, Tahmina': 2,
 'FAULKNALL, Mark': 2,
 'FELSTEAD, Nicholas': 2,
 'FERNANDO, Angelo': 2,
 'FERRIS, Glynn': 2,
 'FIRBANK, Jeremy Kenneth': 2,
 'FLEGG, Scott': 2,
 'FLOUHOUSE, James': 2,
 'FORBES, Jordan': 2,
 'FRAMPTON, Fred': 2,
 'FRANCEY, Tim': 2,
 'FROST, Lee': 2,
 'FUGMANN, Peter Ulrik': 2,
 'FUJIYAMA, Teruo': 2,
 'MURAKAMI, Shuji': 2,
 'FURNESS, Isabel': 2,
 'GABSI, Alexandra': 2,
 'GAIKWAD, Sushil': 2,
 'GARDNER, Amie': 2,
 'GATES, Lee Stuart': 2,
 'GEALS, Ngo Mukum': 2,
 'GIROUX, Guillaume': 2,
 'GIVANS, Sharon Emma': 2,
 'GLEESON, Paul': 2,
 'GOAUDER, Jeannette': 2,
 'GOLDBERGER, David': 2,
 'GOMEZ, Arnaud Clement': 2,
 'GOODMAN-SIMPSON, Paul': 2,
 'GORBAN, Alexandra': 2,
 'GOREN, Erel Antoine': 2,
 'GRAHAM, Thomas': 2,
 'GREASLEY, Sharnie': 2,
 'GREGORY, Phillip Nathan': 2,
 'GRICE, Arthur, Dr': 2,
 'GROSART, Barry': 2,
 'GUEYE, Serigne': 2,
 'HAAFEDH, Harbaoui': 2,
 'ZAHRAN, Hashem': 2,
 'HAINSLEY, Jimell': 2,
 'HAJAMOHIDEEN, Mohamednazeem': 2,
 'HAMILTON, Anthony': 2,
 'HANFI, Farrukh': 2,
 'HANSKI, Nita': 2,
 'HAQ, Abdul Rehman': 2,
 'HARDING, James': 2,
 'HARRIS, Sidney': 2,
 'HARRISON, Ann': 2,
 'HARRISON-BURTON, Thomas Hayden': 2,
 'HARSANT, Clair': 2,
 'HASAN, Metin Fikri': 2,
 'HASELEY, Jamie': 2,
 'HENDRY, Melissa': 2,
 'HERHOLDT, Linda': 2,
 'HESKETH, Emma': 2,
 'HIBBELER, Zhizhen': 2,
 'HIBBERT, Zowie': 2,
 'HILLS, Maureen': 2,
 'HINSLEY, William': 2,
 'HLEIHEL, Mustapha': 2,
 'HOGG, Terrence': 2,
 'HOLLINGSWORTH, Arron': 2,
 'HOLLOWAY, David': 2,
 'HOOLEY, Terence': 2,
 'HOPKINS, James': 2,
 'HOTINCEANU, Augustin Mircea': 2,
 'HU, Keren': 2,
 'HUGHES, Benjamin': 2,
 'HUMPHREYS, James': 2,
 'HUNT, Bernard': 2,
 'HUSSAIN, Adnan': 2,
 'HUSSAIN, Mohsin': 2,
 'HUSSEIN, Cise': 2,
 'IBROUCHEN, Lylia': 2,
 'IDOWU, Ilemobayo': 2,
 'INGLOT, Janusz': 2,
 'ISHARC, Laeeth': 2,
 'ISPAS, Dorel': 2,
 'JACKSON, Paul': 2,
 'JAMES-PERRY, Janice': 2,
 'JANDU, Suneal': 2,
 'JAWAD, Rand': 2,
 'JENKINS, Edward': 2,
 'JENKINS, Luke': 2,
 'JERMYN, Rupert': 2,
 'JHEETA, Manjeet': 2,
 'JOHN BAPTISTE, Glenford': 2,
 'JOHNSON, Francis': 2,
 'JOHNSON, Marcus': 2,
 'JOHNSON, Mark': 2,
 'JOSHI, Rajinder': 2,
 'JOSLIN, Adam': 2,
 'KALLURI, Kartheeka': 2,
 'KANAGASAPATHY, Gayathri': 2,
 'KARNAM, Prabhakar': 2,
 'KASANDA, Richard': 2,
 'KASASA, Moses': 2,
 'KAULBACH, Nick Ian': 2,
 'KAZLOUSKAYA, Alena': 2,
 'KEITA, Marvin': 2,
 'KELLY, Nicole': 2,
 'KENDRICK, Matthew': 2,
 'KENNELL, James': 2,
 'KENT, Calvin': 2,
 'KENTACHE, Atman': 2,
 'KENYON, Richard Pearce': 2,
 'KHALIGHI, Azadeh': 2,
 'KHAN, Farah': 2,
 'KHAN, Khayyam Shezad': 2,
 'KHAN, Mohammed': 2,
 'KHAN, Salma': 2,
 'KHAN, Yassar': 2,
 'KHLIFI, Faouzi': 2,
 'KNIGHT, Adam': 2,
 'KNIGHT, Victoria': 2,
 'KOLEV, Gospodin': 2,
 'KONG, Dae Chul': 2,
 'KRLJES, Sanya, Dr': 2,
 'KROOGLIK, Wilhelmina': 2,
 'KUPERHOLZ, Rael': 2,
 'KUTLU, Taner': 2,
 'KWIATKOWSKI, Sophie': 2,
 'KWIDINI, Anastasia Tabona': 2,
 'LAI, Hsin-Yi': 2,
 'LAINE, Guillaume': 2,
 'LANG, Jing': 2,
 'CAO, Dongpu, Dr': 2,
 'LANGASCHEK, Ionut Cosmin': 2,
 'LAVERY, George Clifford': 2,
 'LAWAL, Taiwo Ademola': 2,
 'LAWRENCE, Andria': 2,
 'LENNOX-NAPIER, Emily': 2,
 ...}